lib/pbench/test/unit/server/conftest.py

```diff
@@ -117,3 +118,21 @@ def ok(user: str) -> str:
         return user
 
     monkeypatch.setattr(Auth, "validate_user", ok)
+
+
+@pytest.fixture
+def db_session(server_config):
+    """
+    Construct a temporary DB session for the test case that will reset on
+    completion.
+
+    NOTE: the client fixture does something similar, but without the implicit
+    cleanup, and with the addition of a Flask context that non-API tests don't
+    require.
```

Comment:
As a reviewer, I mostly appreciate that.  :-D  But, since this function refers to that one, it seems likely that that one should have the complementary explanation referring to this one.